src/PowerShellEditorServices/Language/LanguageService.cs

```diff
@@ -54,6 +55,12 @@ public LanguageService(PowerShellContext powerShellContext)
 
             this.CmdletToAliasDictionary = new Dictionary<String, List<String>>(StringComparer.OrdinalIgnoreCase);
             this.AliasToCmdletDictionary = new Dictionary<String, String>(StringComparer.OrdinalIgnoreCase);
+            this.documentSymbolProviders = new DocumentSymbolProvider[]
```

Comment:
Are you reading my mind?  I was just about to implement a generic "feature provider" model in certain parts of PSES :)